ddp-workspace/projects/sandbox-app/app.yaml

```diff
@@ -0,0 +1,18 @@
+service: sandbox
```

Comment:
Do we actually need to create a deployment for the sandbox?

Reply:
Nah